.github/actions/deploy/action.yaml

```diff
@@ -51,9 +51,11 @@ runs:
     - name: Install operator
       shell: bash
       run: |-
-        set -exu;
+        set -ex;
```

Comment:
should keep the `u` flag, if it causes issue it mean you are using a non-referenced variable at some point...